inc/azure_uamqp_c/connection.h

```diff
@@ -12,13 +12,16 @@
 #include "azure_uamqp_c/amqp_frame_codec.h"
 #include "azure_uamqp_c/amqp_definitions_fields.h"
 #include "azure_uamqp_c/amqp_definitions_milliseconds.h"
+#include "azure_uamqp_c/amqp_definitions_error.h"
+#include "azure_uamqp_c/amqpvalue.h"
 
 #ifdef __cplusplus
 extern "C" {
 #endif /* __cplusplus */
 
     typedef struct CONNECTION_INSTANCE_TAG* CONNECTION_HANDLE;
     typedef struct ENDPOINT_INSTANCE_TAG* ENDPOINT_HANDLE;
+    typedef void* EVENT_SUBSCRIPTION_HANDLE;
```

Comment:
>void* [](start = 12, length = 5)

Should this be a ON_CONNECTION_CLOSED_EVENT_SUBSCRIPTION_TAG*?

Reply:
Thank you Sir. I just realized that my initial idea of having a generic event subscription handle does not really help, it actually takes away the compiler checks

---
In reply to: [192261129](https://github.com/Azure/azure-uamqp-c/pull/240#discussion_r192261129) [](ancestors = 192261129)

Reply:
So, yes, this will be ON_CONNECTION_CLOSED_EVENT_SUBSCRIPTION_TAG

---
In reply to: [192285930](https://github.com/Azure/azure-uamqp-c/pull/240#discussion_r192285930) [](ancestors = 192285930,192261129)